moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -240,6 +240,7 @@ const DashboardView = () => {
           queryRefetch={refetch}
           handleSnackbar={handleSnackbar}
           classes={classes}
+          data={data}
```

Comment:
I will fix the merge conflicts between Tilly's branch and this one, once that gets merged.